lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -2365,6 +2413,11 @@ class ByteCodeBufferReader
         return ReadConstantSizedInt32(buffer, remainingBytes, value);
     }
 
+    const byte * ReadConstantSizedUInt32(const byte * buffer, uint * value)
+    {
+        return ReadConstantSizedInt32(buffer, (int *)value);
```

Comment:
>Int32 [](start = 32, length = 5)

mismatch of ReadUInt32 (name) and ReadInt32 (returned)

Reply:
The uint return is in out param.